draft-packages/collapsible/docs/SingleCollapsible.stories.tsx

```diff
@@ -66,12 +67,27 @@ const SingleCollapsibleCustomHeader = () => (
     variant="default"
     renderHeader={title => (
       <>
-        <Icon icon={translationIcon} />
-        <div style={{ flex: "1 0 auto", marginLeft: "1rem" }}>
-          <Paragraph tag="span" variant="body">
+        <div style={{ flex: "1 0 auto" }}>
```

Comment:
Should these just be in the stylesheet if we already have one? Or is it cool because it's a storybook file

Reply:
I tend to prefer to keep styles inline in stories, otherwise when someone is looking at this story to understand how to use something, they can't easily see what the class is actually doing without digging into the source code.